### PR TITLE
Eslint

### LINKED ISSUE #121: eslint
Currently `eslint` is run as part of mocha using the `test/lint.js`, this is very counter intuitive. Why not run `eslint` on all files and avoid maintaining a `test/lint.js` file?

### PR DESCRIPTION
**Pull request by skovhus**
This resolves #121 

DCO 1.1 Signed-off-by: Kenneth Skovhus kenneth.skovhus@gmail.com

### CODE REVIEW

File: tests/scratch.js [hunk 1]
```diff
@@ -1,25 +0,0 @@
-'use strict';
```
Comment (skovhus) [inline tests/scratch.js]: Remove dead code.

**Comment by DanPurdy** [comment]
I'll leave this for you @Snugug with the 'it was like this when I got here' :smile:

**Comment by Snugug** [comment]
I like the current way as it lets us tie it in to our normal testing suite, but I'm not opposed to this either.

**Comment by Snugug** [comment]
@skovhus In the future, let's have a discussion in the issue queue before you open a PR

**Comment by Snugug** [comment]
Per our [Contributing Guidelines](https://github.com/sasstools/sass-lint/blob/master/CONTRIBUTING.md#submitting-issues)

**Comment by skovhus** [comment]
@Snugug ok. :+1:

**Comment by Snugug** [comment]
Thanks!

**Pull request merged**